src/lib/y2storage/inst_dialog_mixin.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+#
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+Yast.import "Wizard"
+
+module Y2Storage
+  # Mixin providing some simple helpers to deal with dialogs that have special
+  # requirements when used during installation, mainly the expert partitioner.
+  module InstDialogMixin
+    # Runs the given block temporarily disabling the titleOnLeft option that is
+    # by default applied to the installation wizard, so dialogs opened within
+    # the block can use the full screen.
+    #
+    # @example
+    #   dialog = Y2Partitioner::Dialogs.main.new
+    #   dialog_result = without_title_on_left do
+    #     dialog.run
+    #   end
+    def without_title_on_left(&block)
+      Yast::Wizard.OpenNextBackDialog
+      result = block.call
+    ensure
+      Yast::Wizard.CloseDialog
+      result
```

Comment:
this result is skipped.
try it on your own in irb:
```
irb(main):001:0> def a
irb(main):002:1> return 5
irb(main):003:1> ensure
irb(main):004:1* 6
irb(main):005:1> end
=> :a
irb(main):006:0> a
=> 5
```

so it can looks like
```
def without_title_on_left(&block)
￼   Yast::Wizard.OpenNextBackDialog
￼   result = block.call
ensure
  Yast::Wizard.CloseDialog
end
```